docs/src/main/asciidoc/micrometer-metrics.adoc

```diff
@@ -275,7 +275,34 @@ jvm_memory_committed_bytes{area="nonheap",id="CodeHeap 'non-profiled nmethods'",
 Note that metrics appear lazily, you often won't see any data for your endpoint until
 something tries to access it, etc.
 
+== Support for the MicroProfile Metrics API
 
+If you use the MicroProfile Metrics API in your application, the micrometer extension will create an adaptive
```

Comment:
```suggestion
If you use the MicroProfile Metrics API in your application, the Micrometer extension will create an adaptive
```

Reply:
That case mistake is in a few places.. so I'll fix that up.